src/python/pants/backend/java/test/junit_test.py

```diff
@@ -149,12 +154,16 @@ def test_vintage_simple_success(rule_runner: RuleRunner) -> None:
         }
     )
 
+    tgt = rule_runner.get_target(
+        address=Address(
+            spec_path="", target_name="example-test", relative_file_path="SimpleTest.java"
+        )
```

Comment:
Nit: imo the kwargs aren't giving much value

```suggestion
        Address("", target_name="example-test", relative_file_path="SimpleTest.java")
```

Ditto rest of the file

Reply:
I've pulled out a separate function, given how non-DRY the rest of the file was